test.env.example

```diff
@@ -1,5 +1,14 @@
-REDSHIFT_TEST_HOST=
-REDSHIFT_TEST_USER=
-REDSHIFT_TEST_PASS=
-REDSHIFT_TEST_PORT=
-REDSHIFT_TEST_DBNAME=
+# Note: Make sure you have a Redshift account that is set up so these fields are easy to complete.
+
+### Test Environment field definitions
+# These will all be gathered from account information or created by you.
+# Endpoint for Redshift connection
+REDSHIFT_TEST_HOST:
+# Username on your account
+REDSHIFT_TEST_USER:
+# Password for Redshift account
+REDSHIFT_TEST_PASS:
+# Local port to connect on
+REDSHIFT_TEST_PORT:
+# Name of Redshift database in your account to test against
+REDSHIFT_TEST_DBNAME:
```

Comment:
Comparing to the [dbt-snowflake test.env.example](https://github.com/dbt-labs/dbt-snowflake/blob/c086d61650a3f3e6f1733b0d9243603bf566ee90/test.env.example).

Should it be the following instead?
```suggestion
### Test Environment field definitions
# These will all be gathered from account information or created by you.
# Endpoint for Redshift connection
REDSHIFT_TEST_HOST=
# Username on your account
REDSHIFT_TEST_USER=
# Password for Redshift account
REDSHIFT_TEST_PASS=
# Local port to connect on
REDSHIFT_TEST_PORT=
# Name of Redshift database in your account to test against
REDSHIFT_TEST_DBNAME=
```

Reply:
This is a great question and curious to your thoughts on my logic behind this. I went with : over = as I didn't want users  to feel like this was the location for their repository for the variables but to just know correct names to place into a `test.env` they create